plugins/CMakeLists.txt

```diff
@@ -12,85 +11,69 @@ if(OBS_CMAKE_VERSION VERSION_GREATER_EQUAL 3.0.0)
   set_property(GLOBAL APPEND PROPERTY OBS_FEATURES_ENABLED "Plugin Support")
 
   macro(check_obs_browser)
-    if(NOT (OS_WINDOWS AND NOT CMAKE_GENERATOR_PLATFORM MATCHES "(Win32|x64)"))
-      if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser/cmake/macos/Info.plist.in")
-        add_subdirectory(obs-browser)
-      elseif(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser/CMakeLists.txt")
+    if((OS_WINDOWS AND CMAKE_GENERATOR_PLATFORM MATCHES "(Win32|x64)")
+       OR OS_MACOS
+       OR OS_LINUX)
+      if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/obs-browser/CMakeLists.txt")
         message(FATAL_ERROR "Required submodule 'obs-browser' not available.")
+      else()
+        add_subdirectory(obs-browser)
```

Comment:
obs-browser plugins is still missing files
![Screenshot from 2024-01-04 11-31-34](https://github.com/obsproject/obs-studio/assets/17492366/06fa6b71-1a4f-43fd-98ef-e6ca71e064af)

obs-browser-page locale are missing, is this normal ?
![Screenshot from 2024-01-04 11-33-20](https://github.com/obsproject/obs-studio/assets/17492366/b92737af-d2fa-4a08-a227-266c2f1de29a)

Reply:
Where in the original package are `crhome_sandbox` and `vk_swiftshader_icd.json`? They just need to be added to the command copying the other files.

The locale files are not copied for `obs-browser-page` because it doesn't use any - it's just the render process for CEF. It used to ship with the files from `obs-browser` but that was superfluous.

Reply:
> Where in the original package are `chrome_sandbox` and `vk_swiftshader_icd.json`? They just need to be added to the command copying the other files.

`/usr/lib/obs-plugins`, if it was not obvious from the screenshot.